frontend/src/Editor/Components/Table/Table.jsx

```diff
@@ -156,6 +159,10 @@ export function Table({
   const mergeToAddNewRowsDetails = (payload) => dispatch(reducerActions.mergeToAddNewRowsDetails(payload));
   const mounted = useMounted();
 
+  const [showDownloadPopover, setShowDownloadPopover] = useState(false);
+  const [showHideColumnsPopover, setHideColumnsPopover] = useState(false);
```

Comment:
This will re-render the whole table when the popover is toggled. It is better to move the logic to a new/separate component